tf-modules/archive/api.tf

```diff
@@ -20,7 +20,6 @@ locals {
       cmr_username                     = var.cmr_username
       CollectionsTable                 = var.dynamo_tables.collections.name
       DISTRIBUTION_ENDPOINT            = var.distribution_url
-      distributionApiId                = var.distribution_api_id
```

Comment:
I couldn't find this being used anywhere, and it was necessitating passing another TEA output into the cumulus module, so I just removed it

Reply:
I also couldn't find this being used anywhere. Nice catch!